routers/user/home.go

```diff
@@ -307,8 +309,58 @@ func Issues(ctx *context.Context) {
 		return
 	}
 
-	for _, issue := range issues {
-		issue.Repo = showReposMap[issue.RepoID]
+	if !isPullList {
+		for _, issue := range issues {
+			issue.Repo = showReposMap[issue.RepoID]
+		}
+	} else {
+		var repoIDs = make([]int64, 0, len(issues))
+		var shas = make([]string, 0, len(issues))
+		var pullIDs = make([]int64, 0, len(issues))
+		var repoCache = make(map[int64]*git.Repository)
+
+		for _, issue := range issues {
+			issue.Repo = showReposMap[issue.RepoID]
+
+			var rep *git.Repository
+			var ok bool
+			if rep, ok = repoCache[issue.PullRequest.HeadRepoID]; !ok {
```

Comment:
nit: could we call the variable `gitRepo`, or something more descriptive? Without context, it's not obvious (at least not to me) that rep stands for repo.